data/map.txt

```diff
@@ -29167,7 +29283,7 @@ planet Starcross
 planet "Starting Rubin"
 	attributes kimek "coalition station" shipping wealthy
 	landscape land/space3
-	description `Starting Rubin was far from being the first station constructed around the Kimek homeworld, but it is the oldest one to still be in orbit. It was built as an orbital equivalent to the hunger towers found on the planet it orbits, as a safety measure against the disasters that may affect Ki Patek Ka.`
+	description `Starting Rubin was far from being the first station constructed around the Kimek homeworld, but it is the oldest one to still be on orbit. It was built as an orbital equivalent to the hunger towers found on the planet it orbits, as a safety measure against the disasters that may affect Ki Patek Ka.`
```

Comment:
Pretty sure `in orbit` is the correct form.
Google first gives me this `It appears from reading the returns, that 'in orbit' is used to describe the placing of a satellite or object OR to its specific location. 'On Orbit' typically is used to refer to where an action, experiment, or operation is taking place.`

Reply:
I think these are recently fixed typos, which were reintroduced by copying files into the repository

Reply:
Yeah, as Ornok pointed out below, these changes are all due to my forked branch not updating with recent changes made to the continuous branch. Not really sure why that's happening, but I can make manual fixes for this pretty easily.